Removed unsupported versions from database products (#5764)
EnterpriseDB/docs: product_docs/docs/epas/13/troubleshooting/linux_troubleshooting.mdx

> If you encounter difficulty displaying Java-based server features (controls or text not being displayed correctly, or blank windows), upgrading to the latest `libxcb-xlib` libraries should correct the problem on most distributions. Please visit the following link for other possible work-arounds:

Reviewer: ```suggestion
If you encounter difficulty displaying Java-based server features (controls or text not displayed correctly or blank windows), upgrading to the latest `libxcb-xlib` libraries should correct the problem on most distributions. Please visit this [link](http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6532373) for other possible workarounds.
```

> [http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6532373]<http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6532373>

Reviewer: ```suggestion
```
